Procfile

```diff
@@ -0,0 +1 @@
+web: vendor/bin/heroku-php-apache2 public/
```

Comment:
Very well done @rkterungwa16.

Please add an end of line here. You can auto-configure this on every save if you are using VSCode by going to VSCode settings `cmd + ,` and add this to your user settings or workspace settings

```"files.insertFinalNewline": true```

Whenever you save your files, it will auto add a new line at the end of the file.